doc/releases/squid.rst

```diff
@@ -4,6 +4,15 @@ Squid
 
 Squid is the 19th stable release of Ceph.
 
+
+v19.2.2 Squid
+=============
+This is the second hotfix release in the Squid series.
+We recommend that all users update to this release.
+
+Notable Changes
+---------------
+
```

Comment:
Bit sparse here?

Reply:
Yes, we should have a note about what the hotfix is for. Tagging @cbodley

Reply:
https://www.youtube.com/watch?v=y0X0ZYbnHxA